### PR TITLE
Feat/gift card recommendations

### PR DESCRIPTION
**Pull request by NotJohs**
Re [#2565](https://github.com/opencollective/opencollective/issues/2565)

> * Arbitrary list of slugs (only)
>  example `recommendedCollectives: ['babel', 'webpack']`

### CODE REVIEW

File: components/CollectivesWithData.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable graphql/template-strings */
```
Comment (znarf) [inline components/CollectivesWithData.js]: Please don't. Update the GraphQL schema if it's a problem.

File: pages/redeemed.js [hunk 2]
```diff
@@ -276,6 +298,8 @@ class RedeemedPage extends React.Component {
                 <Container maxWidth="1200px">
                   <CollectivesWithData
                     HostCollectiveId={11004} // hard-coded to only show open source projects
+                    // hostCollectiveSlug={'babel'}
+
```
Comment (znarf) [inline pages/redeemed.js]: Remove?

File: pages/redeemed.js [hunk 3]
```diff
@@ -191,6 +191,8 @@ class RedeemedPage extends React.Component {
     const { amount, collective, currency, expiryDate, loading } = this.state;
     const error = this.getError();
     const emitter = this.state.emitter || (data && data.Collective);
+    const recommendedCollectives =
+      get(emitter, 'settings.recommendedCollectives') || get(data.Collective, 'settings.recommendedCollectives');
```
Comment (znarf) [inline pages/redeemed.js]: The second part doesn't look useful? `|| get(data.Collective, 'settings.recommendedCollectives');`

**Comment by NotJohs** [comment]
Screenshot 2019-11-19 at 12 11 58 PM
Screenshot 2019-11-19 at 12 12 09 PM

File: pages/redeemed.js (hunk 3 above)
Comment (NotJohs) [reply pages/redeemed.js]: Yes. 
Sorry, I didn't do a proper clean up.

**Pull request merged**